src/box/memtx_tree.cc

```diff
@@ -2235,19 +2330,21 @@ tree_read_view_create_iterator(struct index_read_view *base,
 }
 
 /** Implementation of create_read_view index callback. */
-template <bool USE_HINT>
+template<bool FAST_OFFSET, bool USE_HINT>
 static struct index_read_view *
 memtx_tree_index_create_read_view(struct index *base)
 {
 	static const struct index_read_view_vtab vtab = {
-		.free = tree_read_view_free<USE_HINT>,
+		.free = tree_read_view_free<FAST_OFFSET, USE_HINT>,
 		.get_raw = tree_read_view_get_raw<USE_HINT>,
```

Comment:
The `tree_read_view_get_raw` needs to consume the `FAST_OFFSET` template parameter too (it's required to implement the read view get in the Tarantool EE (for memtx tree read views).